go.mod

```diff
@@ -103,3 +103,6 @@ replace github.com/pingcap/tidb/parser => ./parser
 
 // fix potential security issue(CVE-2020-26160) introduced by indirect dependency.
 replace github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
+
+// See https://github.com/pingcap/tidb/pull/31010
+replace github.com/tikv/client-go/v2 => github.com/lemonhx/client-go/v2 v2.0.0-rc.0.20211224062518-60bb3ea439ab
```

Comment:
does this mean this PR is blocked by #31010?

Reply:
This is blocked by https://github.com/tikv/client-go/pull/405. The root cause of the compilation error is https://github.com/pingcap/kvproto/pull/828.